stories/components/Card.stories.tsx

```diff
@@ -10,20 +10,20 @@ export default {
 
 const StyledTtitle = styled(Text)`
   color: ${({ theme }) => theme.colors.titles};
-  marginBottom: 0.5rem;
+  marginbottom: 0.5rem;
```

Comment:
Are we sure that works? AFAIK it should be:

```suggestion
  margin-bottom: 0.5rem;
```